modules/hal_nordic/nrfx/Kconfig

```
@@ -66,6 +66,14 @@ config NRFX_GPIOTE
 	bool "Enable GPIOTE driver"
 	depends on HAS_HW_NRF_GPIOTE
 
+config NRFX_GPIOTE_CONFIG_NUM_OF_EVT_HANDLERS
```

```suggestion
config NRFX_GPIOTE_NUM_OF_EVT_HANDLERS
```
for consistency with the `NRFX_CLOCK_LFXO_TWO_STAGE_ENABLED` option and to avoid the double `CONFIG` in the generated macro name.